oauth2_http/java/com/google/auth/oauth2/TokenVerifier.java

```diff
@@ -279,9 +279,6 @@ public TokenVerifier build() {
   /** Custom CacheLoader for mapping certificate urls to the contained public keys. */
   static class PublicKeyLoader extends CacheLoader<String, Map<String, PublicKey>> {
     private static final int DEFAULT_NUMBER_OF_RETRIES = 2;
```

Comment:
curious if there is specific guideline for TokenVerifier to default retry 2 times?

Reply:
I don't think so. I think these values were randomly selected and used. I've kept it the same for now and we can re-visit retries again in the future.